### ISSUE TITLE
Possible bug in gradient clipping of deepq_learner (tf2 branch)

### ISSUE THREAD

**Issue by Giullar** [openai/baselines#1207]
https://github.com/openai/baselines/blob/b99a73afe37206775ac8b884d32a36e213a3fac2/baselines/deepq/deepq_learner.py#L174-L181

In line 179, shouldn't it be:
`grads = clipped_grads`
instead of
`clipped_grads = grads`
?

**Comment by susanth-24** [comment]
yes, I think you are right. It should be grads = clipped_grads, as it is the list containing the gradients after clipping, which should be used for optimization.
